### PR TITLE
Feature/cleanup

### PR DESCRIPTION
**Pull request by jakeonrails**
Getting set to write some bug-fixes here, might as well tidy up a bit while I'm here.

@wesdotcool @otherjonvb

### CODE REVIEW

File: lib/yesmail/subscriber.rb [hunk 1]
```diff
@@ -84,7 +84,7 @@ def api_update
       data_hash = make_hash
       #allowResubscribe must be true when resubscribing a subscriber
       data_hash[:allowResubscribe] = true
-      handler.update(make_hash, path, user_id)
+      handler.update(data_hash, path, user_id)
```
Comment (jakeonrails) [inline lib/yesmail/subscriber.rb:87]: The bug fix! w00t!

File: README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+# Yesmail
```
Comment (wesdotcool) [inline README.md:1]: Awesome. This is just awesome

**Comment by wesdotcool** [comment]
LGTM

**Comment by otherjon** [comment]
Good catch on the bug... but where's the test?

**Pull request merged**